operators/pkg/tenant-controller/tenant_controller_test.go

```diff
@@ -213,11 +213,51 @@ var _ = Describe("Tenant controller", func() {
 			},
 		}
 		Expect(k8sClient.Create(ctx, tn)).Should(Succeed())
+		tnNm := &v1.Namespace{}
+		doesEventuallyExists(ctx, types.NamespacedName{Name: nsName}, tnNm, BeTrue(), timeout, interval)
+		var pvc v1.PersistentVolumeClaim
+		doesEventuallyExists(ctx, types.NamespacedName{Name: UserPvcName, Namespace: tnNm.Name}, &pvc, BeTrue(), timeout, interval)
+
+		pv := &v1.PersistentVolume{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: "pv-test",
+			},
+			Spec: v1.PersistentVolumeSpec{
+				AccessModes: []v1.PersistentVolumeAccessMode{
+					v1.ReadWriteMany,
+				},
+				Capacity: v1.ResourceList{
+					v1.ResourceStorage: *resource.NewQuantity(1*1024*1024*1024, resource.BinarySI),
+				},
+				ClaimRef: &v1.ObjectReference{
+					APIVersion:      "v1",
+					Kind:            "PersistentVolumeClaim",
+					Name:            pvc.Name,
+					Namespace:       pvc.Namespace,
+					ResourceVersion: pvc.ResourceVersion,
+					UID:             pvc.UID,
+				},
+				PersistentVolumeSource: v1.PersistentVolumeSource{
+					CSI: &v1.CSIPersistentVolumeSource{
+						Driver:       "rook-ceph.nfs.csi.ceph.com",
+						VolumeHandle: "path",
+						VolumeAttributes: map[string]string{
+							"server": "rook-ceph-nfs-my-nfs-a",
+							"share":  "/path",
+						},
+					},
+				},
+			},
+		}
+		Expect(k8sClient.Create(ctx, pv)).Should(Succeed())
+		pvc.Spec.VolumeName = "pv-test"
+		Expect(k8sClient.Update(ctx, &pvc)).Should(Succeed())
+		pvc.Status.Phase = v1.ClaimBound
+		Expect(k8sClient.Status().Update(ctx, &pvc)).Should(Succeed())
```

Comment:
Should be moved within/after `checkTnClusterResourceCreation`, possibly in a dedicated function.

Reply:
Moved